front/app/containers/Admin/projects/project/inputImporter/ReviewSection/index.tsx

```diff
@@ -11,9 +11,13 @@ import { useParams } from 'react-router-dom';
 
 import useDeleteIdea from 'api/ideas/useDeleteIdea';
 import useIdeaById from 'api/ideas/useIdeaById';
+import useApproveOfflineIdeas from 'api/import_ideas/useApproveOfflineIdeas';
```

Comment:
Wonder if this should be renamed `useApproveImportedIdeas`?

Reply:
I have a ticket to remove "offline" from everywhere https://www.notion.so/citizenlab/Get-rid-of-word-offline-in-code-and-copies-909988c2cbaf4adfb67abb7dab9b0308